build/redeemer/Redeemer.sol

```diff
@@ -41,6 +42,12 @@ contract Redeemer {
     return true;
   }
 
+  function setLenderAddress(address l) authorized(admin) external returns (bool) {
```

Comment:
nope. we will pass the lender into the `ctor` as we'll deploy lender first. then redeemer (passing lender addr). then marketplace (passing redeemer addr) then set marketpalece address in lender and redeemer